From c5323a4f3d051fe7d752c6c44d91255d72678a8a Mon Sep 17 00:00:00 2001
From: LuK1337 <priv.luk@gmail.com>
Date: Wed, 14 Sep 2016 20:45:04 +0200
Subject: [PATCH 07/15] sepolicy: Allow vold to `getattr` on mkfs_exec

* Fixes denial while trying to format external
  SDcard as adopted storage.

Change-Id: I244ec9b5886888e1cbe488c671b9862b653f73a9
---
 common/private/vold.te | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/common/private/vold.te b/common/private/vold.te
index 628c8c1..4290d17 100644
--- a/common/private/vold.te
+++ b/common/private/vold.te
@@ -1,4 +1,4 @@
 # External storage
-allow vold mkfs_exec:file { execute read open execute_no_trans };
+allow vold mkfs_exec:file { execute read open getattr execute_no_trans };
 allow vold mnt_media_rw_stub_file:dir r_dir_perms;
 allow vold storage_stub_file:dir { rw_file_perms search add_name };
-- 
2.7.4

